pwiz_tools/Skyline/Controls/Graphs/AreaReplicateGraphPane.cs

```diff
@@ -1,1473 +1,1518 @@
+
+            if (Settings.Default.PeakAreaDotpCutoffShow && DotProductDisplayOption.line.IsSet(Settings.Default) && _dotpData != null)
+            {
+                var cutoff = Settings.Default.PeakAreaDotpCutoffValue;
+                var highlightValues = new PointPairList(graphData.DotpData.Select(point => point.Y < cutoff ? point : new PointPair(){X = point.X, Y = float.NaN}).ToList());
+                var cutoffHighlightLine = new LineItem("", highlightValues, Color.DimGray, SymbolType.Circle)
+                {
+                    IsY2Axis = true,
+                    Line = new Line() { Color = Color.Transparent},
+                    Symbol = new Symbol() { Type = SymbolType.Diamond, Size = 9f, Fill = new Fill(Color.Red), Border = new Border(Color.Red, 1) }
+                };
+                CurveList.Insert(0, cutoffHighlightLine);
+                ToolTip.TargetCurves.Add(cutoffHighlightLine);
+
+
+                var belowCutoffCount = _dotpData.Count(dotp => dotp <= cutoff);
+                var labelText = string.Format(Resources.AreaReplicateGraphPane_Replicates_Count_Above_Below_Cutoff,
+                    _dotpData.Count - belowCutoffCount, belowCutoffCount, DotpLabelText);
+                var labelObject = new TextObj(labelText, 1, 0, CoordType.ChartFraction, AlignH.Right, AlignV.Top)
+                {
+                    IsClippedToChartRect = true,
+                    ZOrder = ZOrder.E_BehindCurves,
+                    FontSpec = GraphSummary.CreateFontSpec(Color.Black),
+                };
+                labelObject.FontSpec.Fill = new Fill(Color.Transparent);
+                GraphObjList.Add(labelObject);
+                var cutoffLine = new LineObj()
+                {
+                    IsClippedToChartRect = true,
+                    Location = new Location(0, cutoff, CoordType.XChartFractionY2Scale){Rect = new RectangleF(0, cutoff, 1, 0)},
+                    Line = new LineBase(Color.Red)
+                    };
+                GraphObjList.Add(cutoffLine);
+                //This is a placeholder to make sure the line shows in the legend.
+                CurveList.Insert(CurveList.Count - 1,
+                    new LineItem(String.Format(CultureInfo.CurrentCulture,
+                        Resources.AreaReplicateGraphPane_Dotp_Cutoff_Line_Label, DotpLabelText, cutoff))
+                    {
+                        Points = new PointPairList(new[] { new PointPair(0, 0) }),
+                        Symbol = new Symbol(SymbolType.None, Color.Transparent)
+                    });
+            }
+        }
+
+        public override ImmutableList<float> GetToolTipDataSeries()
+        {
+            return _dotpData;
+        }
+        public override void PopulateTooltip(int index)
+        {
+            ToolTip.ClearData();
+            ToolTip.AddLine(Resources.SummaryReplicateGraphPane_SummaryReplicateGraphPane_Replicate + @":", XAxis.Scale.TextLabels[index]);

```

Comment:
Colons need to go into the RESX text. Japanese and Chinese use a different colon symbol.